reference/orders.v2.oas2.yml

```diff
@@ -4513,9 +4521,9 @@ components:
           type: string
         wrapping_cost_tax_class_id:
           description: |-
-           A read-only value. Do not attempt to set or modify this value in a POST or PUT request.  
-
-           NOTE: Value ignored if automatic tax is enabled on the store.
+            A read-only value. Do not attempt to set or modify this value in a POST or PUT request.  
+            
```

Comment:
Should this line be stripped down to zero whitespace? As per a few of the other spacing lines above and below